storagenode/contact/service.go

```diff
@@ -25,7 +25,8 @@ type Config struct {
 	ExternalAddress string `user:"true" help:"the public address of the node, useful for nodes behind NAT" default:""`
 
 	// Chore config values
-	Interval time.Duration `help:"how frequently the node contact chore should run" releaseDefault:"1h" devDefault:"30s"`
+	//hack: 5s dev default is so the contact chore can send the satellite the capacity info on its second iteration
```

Comment:
I don't think this comment is necessary.

Reply:
Why cannot it send the capacity on the first iteration?

Reply:
If I recall based on a conversation with Jen, I think there is a Monitor chore that must complete at least once before that information is accessible from this service.

Reply:
Can we make contact wait on monitor chore before starting?

Reply:
that is what testplanet does https://github.com/storj/storj/blob/master/internal/testplanet/planet.go#L216, but i think @jenlij may have already tried that with storrj-sim and it didnt work